src/index.js

```diff
@@ -31,4 +31,5 @@ export { default as DialogTitle } from './dialog/DialogTitle'
 export { default as Dropdown } from './Dropdown'
 export { default as Swatch } from './colours/Swatch'
 export { default as TextField } from './TextField'
+export { default as EmailArrayField } from './EmailArrayField'
```

Comment:
Just saw that only the `EmailArrayField` is being exported, while the `ChipArrayField` is not, but both have stories on the storybook.

Reply:
Oh, they both should be, that's an oversight on my part.